hbase-server/src/main/java/org/apache/hadoop/hbase/master/SplitLogManager.java

```diff
@@ -122,9 +121,9 @@ public SplitLogManager(MasterServices master, Configuration conf)
       throws IOException {
     this.server = master;
     this.conf = conf;
-    // Get Server Thread name. Sometimes the Server is mocked so may not implement HasThread.
+    // Get Server Thread name. Sometimes the Server is mocked so may not implement Thread.
     // For example, in tests.
-    String name = master instanceof HasThread? ((HasThread)master).getName():
+    String name = master instanceof Thread? ((Thread)master).getName():
```

Comment:
This is not needed w/ this patch?

Reply:
Seems still needed, since the master here maybe a mocked one.
eg: DummyMasterServices in TestSplitLogManager, it extends MockNoopMasterServices which implement MasterServices, but does not extends Thread.
Thanks.

Reply:
We should not worry about mocked objects. Ideally, we can provide `getName()` default implementation in `MasterServices` and `HMaster` specifically can implement `getName()` as `Thread.getName()`, that way here the caller will just call: `String name = master.getName();`

Reply:
However, there is no harm w/o this change as Stack said. Maybe good to take this clean-up in separate Jira if you would like.
Will merge this PR in a while.

Reply:
Good idea, will fix it in separate jira. 
Thanks.

Reply:
Thanks @bsglz 
I just triggered one build. Let it go through and one result is out, will merge the PR.

Reply:
In addition, maybe we can just always use master.getServerName().toShortString() as name here, WDYT?

Reply:
Yes, is should be fine. Does it match with `((Thread)master).getName()` exactly? If so, let's do it now, else later.

Reply:
Not exactly.
- getName returns master/xxx:16000
- master.getServerName().toShortString() returns xxx:16000
So, will do it later.
Thanks.